pkg/app/app.go

```diff
@@ -148,8 +148,8 @@ func (a *App) InfraConfig() apiv1.InfraConfig {
 }
 
 const (
-	headerNameCOInjectBuildAPICreds = "X-Cutf-Cloud-Orchestrator-Inject-BuildAPI-Creds"
-	headerNameHOBuildAPICreds       = "X-Cutf-Host-Orchestrator-BuildAPI-Creds"
+	headerNameHOBuildAPICreds = "X-Cutf-Host-Orchestrator-BuildAPI-Creds"
```

Comment:
This is an API breaking change, why rename the header?

Reply:
discussed offline, let's keep the two-header solution. will be closing the PR